docs/docs/Releases/release210.md

```diff
@@ -0,0 +1,71 @@
+# Release 2.1.0
+
+Date: 3rd of March, 2023
+
+## Overview
+
+Klaw version 2.1.0 is a minor release with several new features and
+improvements. Key highlights of this release include redesigned approval
+and decline screens in the React UI, Confluent Cloud support, The
+ability to switch between teams, codebase optimizations, and additional
+enhancements.
+
+## What's new in Klaw 2.1.0
+
+### Redesigned Request screens with React UI
+
+With the switch to React, this release features a redesign of the
+following user interfaces:
+
+-   **Approve and Reject Topic Requests**: Users can now approve or
+    reject topic requests using the new UI.
+-   **Approve and Reject Subscription Requests** (Producer/Consumer):
+    Users can now approve or reject ACL requests for both producers and
+    consumers using the new UI.
+-   **Approve and Reject Schema Requests**: Users can now approve or
+    reject schema requests using the new UI.
+
+![image](../../static/images/ApprovalRequestTopic-react.png)
+
+To preview the new Klaw user interface, open the
+`application.properties` file on the Klaw **core** module, and set the
+value of the following property to `true`: :: \# Enable new Klaw user
```

Comment:
The following should be on a new line I missed this the first time I'm sorry, but the note looks perfect now.
# Enable new Klaw user interface 
klaw.coral.enabled=true